sdk/core/core-client/src/createSpan.ts

```diff
@@ -1,59 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { Span, SpanOptions, SpanKind } from "@opentelemetry/api";
-import { getTracer } from "@azure/core-tracing";
-import { OperationOptions, SpanConfig } from "./interfaces";
-
-type OperationTracingOptions = OperationOptions["tracingOptions"];
-
-/**
- * Creates a function called createSpan to create spans using the global tracer.
- * @hidden
- * @param spanConfig - The name of the operation being performed.
- * @param tracingOptions - The options for the underlying http request.
- */
-export function createSpanFunction({ packagePrefix, namespace }: SpanConfig) {
-  return function<T extends OperationOptions>(
-    operationName: string,
-    operationOptions: T
-  ): { span: Span; updatedOptions: T } {
-    const tracer = getTracer();
-    const tracingOptions = operationOptions.tracingOptions || {};
-    const spanOptions: SpanOptions = {
-      ...tracingOptions.spanOptions,
-      kind: SpanKind.INTERNAL
-    };
-
-    const span = tracer.startSpan(`${packagePrefix}.${operationName}`, spanOptions);
-
-    span.setAttribute("az.namespace", namespace);
-
-    let newSpanOptions = tracingOptions.spanOptions || {};
-    if (span.isRecording()) {
-      newSpanOptions = {
-        ...tracingOptions.spanOptions,
-        parent: span.context(),
-        attributes: {
-          ...spanOptions.attributes,
-          "az.namespace": namespace
-        }
-      };
-    }
-
-    const newTracingOptions: OperationTracingOptions = {
-      ...tracingOptions,
-      spanOptions: newSpanOptions
-    };
-
-    const newOperationOptions: T = {
-      ...operationOptions,
-      tracingOptions: newTracingOptions
-    };
-
-    return {
-      span,
-      updatedOptions: newOperationOptions
-    };
-  };
+import {
+  SpanConfig as coreTracingSpanConfig,
+  createSpanFunction as coreTracingCreateSpanFunction
+} from "@azure/core-tracing";
+import { Span } from "@opentelemetry/api";
+import { OperationOptions } from "./interfaces";
+
+export interface SpanConfig extends coreTracingSpanConfig {}
+
+export function createSpanFunction(
```

Comment:
do we need this export anymore? It seems like this change has gone ahead and updated everyone to use `createSpanFunction` from `core-tracing` directly?

Reply:
I'll check now that the smoke is clearing out from all the copy/pastes I did. :)

Reply:
The answer was 'no'. Removed and now the only other spot that exports telemetry related things is core-http, and all of those exports are now marked deprecated (and not used, now, by any packages in master).